src/components/progress/progress.test.tsx

```diff
@@ -0,0 +1,47 @@
+import { beforeEach, describe, expect, test } from "vitest";
+import { render, screen } from "@testing-library/react";
+import Progress from "./progress"; // Adjust the import path as needed
+
+describe("<Progress/>", () => {
+  beforeEach(() => {
+    render(<Progress />);
+  });
+
+  test("renders the title", () => {
+    const heading = screen.getByRole("heading", { level: 1 });
+    expect(heading).toHaveTextContent("Progress");
+  });
+
+  test("renders the profile information", () => {
+    const profileSection = screen.getByRole("heading", {
+      level: 2,
+      name: /Profile Section/i,
+    });
+    expect(profileSection).toBeInTheDocument();
+
+    expect(screen.getByRole("heading", { level: 3 })).toHaveTextContent(
+      "0 Attempting",
+    );
+    expect(screen.getByText(/Easy: 12\/12/i)).toBeInTheDocument();
+    expect(screen.getByText(/Med.: 22\/22/i)).toBeInTheDocument();
+    expect(screen.getByText(/Hard: 1\/1/i)).toBeInTheDocument();
```

Comment:
만약 정규식으로 테스트를 하고싶다면 숫자부분 수정과 i태그를 수정요청드립니다
```suggestion
    expect(screen.getByText(/^Easy: \d{1,2}\/\d{1,2}$/)).toBeInTheDocument();
    expect(screen.getByText(/^Med.: \d{1,2}\/\d{1,2}$/)).toBeInTheDocument();
    expect(screen.getByText(/^Hard: \d{1,2}\/\d{1,2}$/)).toBeInTheDocument();
```
- ^ $ 를통해서 정해진 format작성
- 숫자부분은 총 문제가 75문제 이하니 2자리를 못넘기기에 2자리까지 제한
- i태그는 정확한 string비교를 위해 제거 https://ko.javascript.info/regexp-introduction

 @SamTheKorean 님은 어떤 방법이 테스트하기에 용이하다고 생각하시나요?

Reply:
어떤 테스트를 할 지 고민했었는데 추천하신 것처럼 정규식에 제한을 두는 것 좋은 생각인 것 같습니다! 수정하겠습니다.